### PR TITLE
fix(css): use google cdn for opensans fonts

### PR DESCRIPTION
**Pull request by jiekang**
This addresses https://openshift.io/openshiftio/Openshift_io/plan/detail/156

The OpenSans fonts are downloaded from CDN instead of being packaged in our distribution. A local prod build via `npm run build:prod` has a reduction of ~13 MB, from 36 MB to 23 MB on my machine.

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @jiekang for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by jiekang** [comment]
At this time `pre-build.sh` exists to create the fontless css/less Patternfly files used in the project. An issue has been opened for Pattiernfly asking if they can package these in their distribution. Otherwise I'll be trying to run these 'pre-build' in the npm scripts.

File: post-install.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Removes first line of patternfly.min.css : This assumes this is the line containing the Open Sans font definitions
+tail -n +2 "node_modules/patternfly/dist/css/patternfly.min.css" > "node_modules/patternfly/dist/css/patternfly.fontless.min.css"
```
Comment (andrewazores) [inline post-install.sh]: I don't really like this "remove the first line" approach... couldn't a similar `sed` trick as below be done for this step, too?

Comment (jiekang) [reply post-install.sh]: Yeah good point. I've addressed it now

**Comment by alien-ike** [comment]
Hey @andrewazores! It seems you tried to trigger `/ok-without-tests` command, but this will not have any effect due to insufficient permission. You have to be admin or requested reviewer or pull request approver, but not pull request creator for this command to take an effect.

**Comment by andrewazores** [comment]
I'll test this locally but I think it seems reasonable from a reading.

**Comment by jiekang** [comment]
Interesting failure from CICO:
```
++ cico node get -f value -c ip_address -c comment
The requested operation failed as no inventory is available.
+ cico_output=
+ '[' 1 -eq 0 ']'
+ '[' 6 -gt 5 ']'
+ echo 'giving up on '\''cico node get'\'''
giving up on 'cico node get'
+ exit 1
Build step 'Execute shell' marked build as failure
```

**Pull request merged**